code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -104,6 +104,14 @@ USE THIS CHEMISTRY DISPENSER FOR MAPS SO THEY START AT 100 ENERGY
 			stat |= NOPOWER
 	nanomanager.update_uis(src) // update all UIs attached to src
 
+/obj/machinery/chem_dispenser/proc/can_use(var/mob/living/silicon/robot/R)

```

Comment:
I'm not sure all these can_use procs are working as you intend, did you make sure this would actually allow mommis to use it?